pinot-record-readers/pinot-avro/pom.xml

```diff
@@ -42,6 +42,33 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>build-shaded-jar</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-shade-plugin</artifactId>
+            <version>3.2.1</version>
+            <executions>
+              <execution>
+                <phase>package</phase>
+                <goals>
+                  <goal>shade</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>target/plugins/record-reader/avro-record-reader-package</outputDirectory>
```

Comment:
can we change it to plugins/record-readers/pinot-avro or plugins/record-readers/pinot-avro-pkg  match the module name.

Reply:
will do

Reply:
i prefer just use `pinot-avro`